setup.py

```diff
@@ -38,7 +38,7 @@
     # NOTE: Apache Beam tests depend on this library and cannot
     # currently upgrade their httplib2 version.
     # Please see https://github.com/googleapis/google-api-python-client/pull/841
-    "httplib2>=0.9.2,<1dev",
+    "httplib2>=0.15.0,<1dev",
```

Comment:
@aaltay Is it alright to bump this now?

Reply:
I _believe_ it can be bumped now. Beam has `'httplib2>=0.8,<0.18.0',` as the new range now. (https://github.com/apache/beam/blob/8bedfd402ccd1336f2b40fad58cd5833cecb9638/sdks/python/setup.py#L150)

Do you remember what was the specific problem?

(/cc @tvalentyn in case he remembers.)

Reply:
Yes, no need to restrict this for Beam reasons anymore.
https://issues.apache.org/jira/browse/BEAM-9819 has some context for prior restrictions.

Reply:
Thanks!

Reply:
@busunkim96 Done!